### ISSUE TITLE
[How-To] Move taskbar buttons w/ left mouse button: restore previous behaviour

### ISSUE THREAD

**Issue by msamesch** [bbidulock/icewm#624]
Is there an easy way to restore the previous behaviour, i.e. move taskbar buttons with the right mouse button. Otherwise, could someone please point me to the corresponding location in the sources? I'm using the double height taskbar.

TIA

**Comment by gijsbers** [comment]
Thanks! It still works fine here. There shouldn't be any change.
What was your previous situation and why did you change it?
Is there no response at all when you attempt to drag a task button?

**Comment by msamesch** [comment]
Thanks for your reply.

You're welcome. I've been using IceWM for quite some time now and I think it is just great. :+1: 


With Debian Stretch I used the right mb to drag task buttons, but after upgrading to Devuan Beowulf lately, I can drag them with the left mb. So if the behaviour is supposed to be unchanged, it might be a problem with my system.

Maybe the problem is solved with the upgrade Chimaera. I'll check this tomorrow.

Edit: The IceWM version in Stretch is from 2016 (1.3.8+mod+20161220-1). I should have mentioned this right at the beginning. ;)

**Comment by gijsbers** [comment]
Could you tell us the icewm version numbers involved.
Those OS release names don't prove much.

**Comment by msamesch** [comment]
My 'Edit' was too late. ;)
Now it's at version 1.4.3.0~pre-20181030-2

**Comment by gijsbers** [comment]
1.3.8 is very very old. 1.4.3 is only slightly less old. We are now at 2.9.0. At least twice as good!
The website ice-wm.org explains how to compile from this repository. I can recommend that!

**Comment by msamesch** [comment]
Yes, my system was (and still is) 'a little' outdated. ;) Next would be 2.1.2, so still rather old. Building and installation looks straight forward. I'm curious to see the latest version.

**Comment by rubyFeedback** [comment]
> Building and installation looks straight forward.

Icewm is really easy to compile even on older linux systems IMO. I even think it is simpler to compile correctly than fluxbox these days (with vanilla fluxbox compiled from source I tend to have the problem that I can not re-size the windows; other compiled fluxbox versions do not have that issue when I use them, so something must be broken on my setup. Icewm on the other hand compiles nicely and then has no issues running it; and even the very few bugs I reported, gijsbers was able to quickly fix so I venture a guess that icewm's code base is in a better state than fluxbox).

Make sure that the base requirements are setup correctly on your system, such as imlib2 or cmake. I have had issues related to cmake from default distributions, in particular the dread -fPIC (or was it -fPIE) "can not relocate static xyz" or something. I did not have that issue when I compiled cmake from source into /usr/. I don't know why most linux distributions don't really work that well by default, always requiring the user to fix things on his/her own, or compile from source. Slackware is one of the few exceptions here; after I install it, it really works very, very well. Oldschool for the win! \o/
